spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/mysql/V3__Add_Task_Metadata_Table.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.db.migration.mysql;
+
+import java.util.Arrays;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+
+import org.springframework.cloud.dataflow.server.db.migration.SqlCommand;
+import org.springframework.cloud.dataflow.server.db.migration.SqlCommandsRunner;
+
+/**
+ * Adds the database table to store task manifests in (one for each execution).
+ *
+ * @author Michael Minella
+ * @since 2.3
+ */
+public class V3__Add_Task_Metadata_Table extends BaseJavaMigration {
+
+	public final static String CREATE_TASK_METADATA_TABLE =
+			"CREATE TABLE task_execution_metadata (\n" +
+					"    id BIGINT NOT NULL,\n" +
+					"    task_execution_id BIGINT NOT NULL,\n" +
+					"    task_execution_manifest TEXT,\n" +
```

Comment:
In other parts we've used `longtext` instead of `text` as a type.

Reply:
Done